content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +162,67 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+For every `dvc.yaml` file, a corresponding `dvc.lock` (YAML) file is created or
+updated by certain DVC commands (`dvc run`, `dvc repro`, `dvc commit`). It
+describes the latest contents of the files involved in the pipeline(s). It has
+several purposes:
+
+- Basic tracking of intermediate and final results of a pipeline (similar to
+  [`.dvc` files](#dvc-files))
+- Allow DVC to detect when stage definitions, or their dependencies have
+  changed. Such conditions invalidate states, requiring their reproduction (see
+  `dvc status`, `dvc repro`).
+- Provides a mapping of file and directory paths tracked in the
```

Comment:
this sounds like a part of the data tracking (fist point in the list), or I'm missing something

Reply:
No, you're right. It's repetitive. The intention of this bullet was to say that it's needed by a bunch of commands to work, let me try to rewrite these 2 bullets... ⏳

Reply:
Updated in 9638a131. PTAL